### PR TITLE
Made background transparent

### PR DESCRIPTION
**Pull request by mangalgithub**
Made the background transparent of login and signup form.

### CODE REVIEW

File: blog/migrations/0001_initial.py
Comment (kunal2812) [inline blog/migrations/0001_initial.py:1]: Remove this file

**Comment by kunal2812** [comment]
@mangalgithub Add description to the PR.

**Comment by mangalgithub** [comment]
Description
Made background of login and signup transparent.

models:



Functionality that on clicking on signup button the login section become transparent and signup section become active and vice versa.


Related Issue
Fixes https://github.com/ContriHUB/Sheher/issues/59

Type of change
 Bug fix (non-breaking change which fixes an issue)
 New feature (non-breaking change which adds functionality)

How Has This Been Tested?
There are no errors after the implementation.

Screenshots (if appropriate):
Checklist:
 I have registered myself at [Contrihub website](https://sac.mnnit.ac.in/contrihub).
 My code follows the code style of this project.
 I have commented my code, particularly in hard-to-understand areas
 My changes generate no new warnings
 I have read the CONTRIBUTING document.
 All new and existing tests passed.
Test Configuration:

Firmware version:
Hardware:
Toolchain:
SDK:

**Comment by kunal2812** [comment]
image

This button is missing from your changes

image

Fix CSS here.

**Review by kunal2812 — APPROVED** [review]
Looking great

**Pull request merged**